### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thelovekesh**
Fixes some writing issues.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Some of the changes don't seem right, so please @mdo @patrickhlauke have a look.

File: README.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ Have a bug or a feature request? Please first read the [issue guidelines](https:
 
 Bootstrap's documentation, included in this repo in the root directory, is built with [Hugo](https://gohugo.io/) and publicly hosted on GitHub Pages at <https://v5.getbootstrap.com/>. The docs may also be run locally.
 
-Documentation search is powered by [Algolia's DocSearch](https://community.algolia.com/docsearch/). Working on our search? Be sure to set `debug: true` in `site/assets/js/src/search.js` file.
+The documentation search is powered by [Algolia's DocSearch](https://community.algolia.com/docsearch/). Working on our search? Be sure to set `debug: true` in the `site/assets/js/src/search.js` file.
```
Comment (patrickhlauke) [inline README.md:131]: the addition of the second "the" here is probably the only one i'd say yes, it's a good change. all others i'd say are not essential/matter of opinion/not correct

Comment (XhmikosR) [reply README.md:131]: Feel free to dismiss it. I don't want to mark this as invalid since it doesn't seem like the other PRs we got so far :)

**Comment by mdo** [comment]
This is fine as-is.

**Pull request closed without merging**